lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriters.java

```diff
@@ -202,23 +212,27 @@ private static PrimitiveValueWriter getValueWriter(ValuesWriter valuesWriter, Ty
             verifyParquetType(type, parquetType, OriginalType.TIME_MICROS);
             return new TimeMicrosValueWriter(valuesWriter, parquetType);
         }
-        if (TIMESTAMP_MILLIS.equals(type)) {
-            verifyParquetType(type, parquetType, OriginalType.TIMESTAMP_MILLIS);
-            // TODO when writing with Hive connector, isAdjustedToUTC is being set to true, which might be incorrect
-            //   verifyParquetType(type, parquetType, TimestampLogicalTypeAnnotation.class, isTimestamp(LogicalTypeAnnotation.TimeUnit.MILLIS));
-            return new TimestampMillisValueWriter(valuesWriter, type, parquetType);
-        }
-        if (TIMESTAMP_MICROS.equals(type)) {
-            verifyParquetType(type, parquetType, OriginalType.TIMESTAMP_MICROS);
-            // TODO when writing with Hive connector, isAdjustedToUTC is being set to true, which might be incorrect
-            //   verifyParquetType(type, parquetType, TimestampLogicalTypeAnnotation.class, isTimestamp(LogicalTypeAnnotation.TimeUnit.MICROS));
```

Comment:
Per https://github.com/trinodb/trino/pull/12857#discussion_r902579834 should the application of this TODO be a separate commit?